test/e2e/run_entrypoint_test.go

```diff
@@ -28,6 +28,17 @@ var _ = Describe("Podman run entrypoint", func() {
 
 	})
 
+	It("podman run no command, entrypoint, or cmd", func() {
+		dockerfile := `FROM docker.io/library/alpine:latest
+ENTRYPOINT []
+CMD []
+`
+		podmanTest.BuildImage(dockerfile, "foobar.com/entrypoint:latest")
+		session := podmanTest.Podman([]string{"run", "foobar.com/entrypoint:latest"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(125))
+	})
```

Comment:
consider adding a test here for the number of containers, s/b zero.  Don't hold the submit for that if you decide to do it.